scripts/data/csv/index.ts

```diff
@@ -18,14 +18,8 @@ export function buildingNumberFromId(buildingId: string) {
 }
 
 export function getData(csvPath: string) {
-  return new Promise<Record<string, any>[]>((resolve, reject) => {
-    readFile(csvPath, "utf8", (err, data) => {
-      if (err) {
-        return reject(err);
-      }
-      resolve(parse(data, parserOptions) as Record<string, any>[]);
-    });
-  });
+  return fs.readFile(csvPath)
+    .then((data) => parse(data, parserOptions))
```

Comment:
Best change ever! I wonder though what's the reason to remove type coercion?

Reply:
My editor didn't show any errors from removing it but I can add it back if you want to, unsure what's its use is.